app/models/contestant.rb

```diff
@@ -17,7 +17,8 @@
 #
 
 class Contestant < ActiveRecord::Base
-  has_and_belongs_to_many :seasons
+  has_many :contestants_seasons, class_name: 'ContestantsSeasons'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.